src/lightning_habana/pytorch/strategies/deepspeed.py

```diff
@@ -247,6 +154,140 @@ def __init__(
         precision_plugin: Optional[PrecisionPlugin] = None,
         process_group_backend: Optional[str] = "hccl",
     ) -> None:
+        """Strategy to support deepspeed with HPU devices.
```

Comment:
Was this need to move? unfortunately with git is hard to track if there are some other changes than identical copy...

Reply:
this was for precommit errors